serpentTools/objects/containers.py

```diff
@@ -0,0 +1,92 @@
+import numpy
```

Comment:
The `numpy` module isn't used here, so this line can be removed.

Further, could you add a brief [module level docstring](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#documenting-modules) at the top of this file? It largely just needs to indicate the purpose of the file and classes present (to be expanded as we go)

Reply:
Done